core/client/src/test/java/alluxio/client/file/policy/DeterministicHashPolicyTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file.policy;
+
+import alluxio.Constants;
+import alluxio.client.block.BlockWorkerInfo;
+import alluxio.client.file.policy.options.BlockLocationPolicyCreateOptions;
+import alluxio.client.file.policy.options.BlockLocationPolicyGetWorkerOptions;
+import alluxio.wire.WorkerNetAddress;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * Tests {@link DeterministicHashPolicy}.
+ */
+public final class DeterministicHashPolicyTest {
+  private static final int PORT = 1;
+
+  private List<BlockWorkerInfo> mWorkerInfos;
+
+  @Before
+  public void before() {
+    mWorkerInfos = new ArrayList<>();
+    mWorkerInfos.add(new BlockWorkerInfo(new WorkerNetAddress().setHost("worker1")
+        .setRpcPort(PORT).setDataPort(PORT).setWebPort(PORT), Constants.GB, 0));
+    mWorkerInfos.add(new BlockWorkerInfo(new WorkerNetAddress().setHost("worker2")
+        .setRpcPort(PORT).setDataPort(PORT).setWebPort(PORT), 2 * (long) Constants.GB, 0));
+    mWorkerInfos.add(new BlockWorkerInfo(new WorkerNetAddress().setHost("worker3")
+        .setRpcPort(PORT).setDataPort(PORT).setWebPort(PORT), 3 * (long) Constants.GB, 0));
+    mWorkerInfos.add(new BlockWorkerInfo(new WorkerNetAddress().setHost("worker4")
+        .setRpcPort(PORT).setDataPort(PORT).setWebPort(PORT), 3 * (long) Constants.GB, 0));
+  }
+
+  @Test
+  public void getWorkerDeterministically() {
+    DeterministicHashPolicy policy = (DeterministicHashPolicy) BlockLocationPolicy.Factory.create(
+        BlockLocationPolicyCreateOptions.defaults()
+            .setLocationPolicyClassName("alluxio.client.file.policy.DeterministicHashPolicy"));
```

Comment:
`DeterministicHashPolicy.class.getName()` to be more flexible for code refactor?